tests/tbasic

```diff
@@ -150,7 +149,25 @@ if [ $FAIL -ne 0 ]; then
 fi
 
 OPENSSL_CONF=${ORIG_OPENSSL_CONF}
-rm -f "${OPENSSL_CONF}.nopin"
+
+title PARA "Test EVP_PKEY_eq on public RSA key both on token"
+$CHECKER ./tcmpkeys "$PUBURI" "$PUBURI"
+title PARA "Test EVP_PKEY_eq on public EC key both on token"
+$CHECKER ./tcmpkeys "$ECPUBURI" "$ECPUBURI"
+
+title PARA "Test EVP_PKEY_eq on public RSA key via import"
+$CHECKER ./tcmpkeys "$PUBURI" "${TMPPDIR}"/rsa.pub.uripin.pem
+title PARA "Test EVP_PKEY_eq on public EC key via import"
+$CHECKER ./tcmpkeys "$ECPUBURI" "${TMPPDIR}"/ec.pub.uripin.pem
+
+title PARA "Test EVP_PKEY_eq with key exporting disabled"
+ORIG_OPENSSL_CONF=${OPENSSL_CONF}
+OPENSSL_CONF=${OPENSSL_CONF}.noexport
+title PARA "Test RSA key"
+$CHECKER ./tcmpkeys "$PUBURI" "$PUBURI"
+title PARA "Test EC key"
+$CHECKER ./tcmpkeys "$ECPUBURI" "$ECPUBURI"
```

Comment:
We should test also comparison of the EdDSA keys if possible

Reply:
maybe open an issue for that?